### ISSUE TITLE
replication connection: bouncer config error

### ISSUE THREAD

**Issue by thamerlan** [pgbouncer/pgbouncer#1196]
Trying to setup a logical replication with:
`CREATE SUBSCRIPTION sub_tlu_test CONNECTION 'host=<pgbouncer_ip_here> port=11432 dbname=mydb sslmode=disable user=sub_logical_test password=abc123'
         PUBLICATION pub_tlu_test
         WITH (slot_name=test_slot_v1, create_slot=false);
`
which fails with:

In the pgboucer log: 
`WARNING pooler error: bouncer config error` 
(debug -vvv gives no additional error messages)

In the target db (publication) log:
`ERROR:  extended query protocol not supported in a replication connection`

P.s.
`PgBouncer 1.23.1
libevent 2.1.8-stable
adns: evdns2
tls: OpenSSL 1.1.1  11 Sep 2018
Ubuntu 18.04 LTS
PostgreSQL 14.13
`

bouncer config:
```[databases]
mydb = host=10.10.10.5 dbname=mydb port=5432 auth_user=postgres

[pgbouncer]
pool_mode = session
max_client_conn = 200
default_pool_size = 100
reserve_pool_timeout = 2
reserve_pool_size = 100

log_disconnections = 0
log_pooler_errors = 1

server_reset_query = DISCARD ALL;

server_check_delay = 180
server_lifetime = 900
server_idle_timeout = 180

listen_port = 11432
listen_addr = *

logfile = /<path to file here>
pidfile = /<path to file here>
admin_users = postgres
stats_users = postgres
; ignore_startup_parameters = extra_float_digits, options
; application_name_add_host = 1

auth_type = md5
auth_query = SELECT usename, passwd FROM pg_shadow WHERE usename=$1
auth_file = /<path to file here>```

**Comment by clintonb** [comment]
@thamerlan what did you learn? How did you resolve this issue?

**Comment by thamerlan** [comment]
> [@thamerlan](https://github.com/thamerlan) what did you learn? How did you resolve this issue?

Don't use `auth_query `, use `auth_file`. 
But looks like it was fixed in the latest `1.24.0` release: `Fix bug when using auth_query and replication connections together. This bug would cause connection failures in such setups. `

**Comment by clintonb** [comment]
Thanks. In my case I forgot to configure the replica (Cloud SQL Postgres) to grant my bouncer user access to the auth tables and pg_shadow.
